spec/requests/form_controller_spec.rb

```diff
@@ -746,7 +746,6 @@
           it "re-renders the same page with errors if validation fails" do
             post "/lettings-logs/#{lettings_log.id}/managing-organisation", params: params
             expect(page).to have_content("There is a problem")
-            expect(page).to have_content("Error: Which organisation manages this letting?")
```

Comment:
And here? Are these failing? cause I don't think they should

Reply:
Ye it was failing, I think because they're testing pre-2023/2024 so don't use the translation files.

Reply:
Updated most of these tests to 2023, so the labels now can be tested.

Reply:
Nice, thanks!